src/NLog/Config/XmlLoggingConfiguration.cs

```diff
@@ -336,12 +336,12 @@ private void Initialize(XmlReader reader, string fileName, bool ignoreErrors)
                     }
                     else
                     {
-                        InternalLogger.Error("Error in Parsing Configuration File. Exception : {0}", ConfigException);
+                        InternalLogger.Error(ConfigException, "Error in Parsing Configuration File.");
```

Comment:
In fact, is the condition `if (exception.MustBeRethrown())` not supposed to say `if (exception.MustBeRethrownImmediately())`? (in which case my previous comment would no longer be applicable).

Is this PR not effectively renaming the old `MustBeRethrown()` to `MustBeRethrownImmediately()`.